Fusion latest new accounts (#8815)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

- **Enhancement**: The Model Timing tab displays an informative banner for <Constant name="fusion_engine" /> runs instead of the timing chart. The banner explains "Model timing is not yet available for Fusion runs" and provides context about threading differences. Non-Fusion runs continue to show the timing chart normally.
- **Behavior change**: [Snowflake plans to increase](https://docs.snowflake.com/en/release-notes/bcr-bundles/un-bundled/bcr-2118) the default column size for string and binary data types in May 2026. `dbt-snowflake` versions below v1.10.6 may fail to build certain incremental models when this change is deployed. [Assess impact and take any required actions](/reference/resource-configs/snowflake-configs#assess-impact-and-required-actions).
- **New**: The new <Constant name="semantic_layer"/> YAML specification is now available on the <Constant name="dbt_platform" /> **Latest** release track. For an overview of the changes and steps how to migrate to the latest YAML spec, see [Migrate to the latest YAML spec](/docs/build/latest-metrics-spec).

> - **Behavior change:** New projects in trial, starter, or Enterprise accounts now default to **Fusion Latest** for new deployment environments when a supported adapter (Redshift, Snowflake, BigQuery, and BigQuery) is selected. You can revert to another version by changing the dbt version in your [environment settings](/docs/dbt-cloud-environments#change-an-environment).

Author: ```suggestion
- **Behavior change:** New projects in trial, starter, or Enterprise accounts now default to **Fusion Latest** for new deployment environments when a supported adapter (Redshift, Snowflake, BigQuery, and Databricks) is selected. You can revert to another version by changing the dbt version in your [environment settings](/docs/dbt-cloud-environments#change-environment-settings).
```

> - **Behavior change:** New projects in trial, starter, or Enterprise accounts now default to **Fusion Latest** for new deployment environments when a supported adapter (Redshift, Snowflake, BigQuery, and Databricks) is selected. You can revert to another version by changing the dbt version in your [environment settings](/docs/dbt-cloud-environments#change-environment-settings).

Reviewer: It defaults for all environments (include development) so not just deployment envs

> - **Behavior change:** New projects in trial, starter, or Enterprise accounts now default to **Fusion Latest** for new deployment environments when a supported adapter (Redshift, Snowflake, BigQuery, and Databricks) is selected. You can revert to another version by changing the dbt version in your [environment settings](/docs/dbt-cloud-environments#change-environment-settings).

Author: ```suggestion
- **Behavior change:** New projects in trial, starter, or Enterprise accounts now default to **Fusion Latest** for all new environments with a supported adapter (Redshift, Snowflake, BigQuery, and Databricks). You can revert to another version by changing the dbt version in your [environment settings](/docs/dbt-cloud-environments#change-environment-settings).
```
